packages/link/README.md

```diff
@@ -105,6 +101,25 @@ When a link needs to be placed on top of a colored background or a visual, use t
 </div>
 ```
 
+### Quiet links
+
+All links can have a quiet style, which means they don’t have an underline. This style should only be used when the placement and context of the link is explicit enough that a visible underline isn’t necessary. Quiet links are less accessible, so they should not be used for links that are essential to the experience. These are commonly used in website footers, where there are several lists of links that are shortcuts to other pages.
+
+<!-- prettier-ignore -->
+```html
+<p>This is a <sp-link quiet href="#">quiet standard link</sp-link>.</p>
+<p>This is a <sp-link quiet variant="secondary" href="#">quiet secondary link</sp-link>.</p>
+<div
+    style="background-color: #0f797d; padding: 15px 20px; display: inline-block;"
+>
+    <p style="color: rgb(240, 240, 240);">
+        This is a
+        <sp-link over-background quiet href="#">quiet link</sp-link>
+        is over a background.
```

Comment:
I think this is intended to be, "This is a quiet link over a background," or, "This quiet link is over a background."